medusa/process_tv.py

```diff
@@ -24,7 +24,7 @@
 from six import iteritems
 
 
-log = BraceAdapter(logging.getLogger(__name__))
+log = CustomBraceAdapter(logging.getLogger(__name__))
```

Comment:
@medariox the way this works, is. CustomBraceAdapter is a subclass of BraceAdapter. Only in CustomBraceAdapter the log message (only the message not the kwargs) is evaluated. And if found, the log level is overwritten (only if set). 0 == unset.

But because we evaluate the message (and not the parsed message), I had to refactor process_tv.py. (which I think is not a bad thing). I removed the "old" medusa.logger('lalal', logger.WARNING). And at the same time improved the self.log() method a little, so both logging and format use the **kwargs.

Reply:
I renamed self.log to self.log_and_output, to eliminate confusing between sef.log() and log.log()